### DISCUSSION TITLE
What does the git push command do?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#238] [Q&A] [answered]
Short answer.

**Comment by montasim** [comment, marked as answer]
The [Git push command](https://www.simplilearn.com/tutorials/git-tutorial/git-push-command) is used to push the content in a local repository to a remote repository. After a local repository has been modified, a push is executed to share the modifications with remote team members.

![image](https://user-images.githubusercontent.com/95298623/197933387-2be82dcd-5a7d-46bb-a35d-821510d2b643.png)
